packages/next/build/index.ts

```diff
@@ -401,7 +401,10 @@ export default async function build(dir: string, conf = null): Promise<void> {
       }
 
       plugin.ampPages.forEach((pg: any) => {
-        pageInfos.get(pg)!.isAmp = true
+        // Server-side does not compile .es6 version
+        if (!pg.includes('.module')) {
```

Comment:
I'm skeptical that this is in the correct place, but I can't suggest a better location for it right now. Let me look around.

Reply:
Moved this behavior to the plugin.